packages/thirdweb/src/extensions/lens/read/resolveAddress.ts

```diff
@@ -0,0 +1,106 @@
+import type { Chain } from "../../../chains/types.js";
+import type { ThirdwebClient } from "../../../client/client.js";
+import { getAddress, isAddress } from "../../../utils/address.js";
+
+/**
+ * @extension LENS
+ */
+export type ResolveLensAddressParams = {
+  /**
+   * Either a full lens handle, or a lens localName
+   * Examples:
+   * 1. Full handle: "lens/vitalik"
+   * 2. Just local name: "vitalik"
+   */
+  name: string;
+  /**
+   * a [`ThirdwebClient`](https://portal.thirdweb.com/references/typescript/v5/ThirdwebClient)
+   */
+  client: ThirdwebClient;
+  /**
+   * Override parameters for Lens contract, defaults to LensHandle contract on Polygon
+   */
+  overrides?: {
```

Comment:
do we really need overrides nesting here? just add them inline imo since they're optional

Reply:
So in the other Lens methods that have more than one override-able contracts, I group them all into an object to signify that "all these contracts have to be on the same chain".

For this one I keep the same pattern for consistency - but I'm okay with either way

Reply:
Ok that makes sense